packages/proverifdoc/proverifdoc.2.05/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "2.0"
+synopsis: "Documentation for ProVerif, a cryptographic protocol verifier in the symbolic model"
+maintainer:
+  "Bruno Blanchet <bruno.blanchet@inria.fr>, Vincent Cheval <vincent.cheval@inria.fr>"
+authors:
+  "Bruno Blanchet <bruno.blanchet@inria.fr>, Vincent Cheval <vincent.cheval@inria.fr>, Ben Smyth <research@bensmyth.com>, Marc Sylvestre"
+homepage: "http://proverif.inria.fr/"
+dev-repo: "git+https://gitlab.inria.fr/bblanche/proverif.git"
+bug-reports: "proverif-dev@inria.fr"
+license: "Public domain"
```

Comment:
The CI linter check complains that "Public domain" is not listed in https://spdx.org/licenses. But it has been this way so I don't believe this is a blocker.

```
warning 62: License doesn't adhere to the SPDX standard, see https://spdx.org/licenses/: "Public domain"
```

This PR looks good to merge for me.